lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3112,32 +3112,24 @@ LogicalResult StmtEmitter::visitSV(AliasOp op) {
 
 LogicalResult StmtEmitter::visitSV(InterfaceInstanceOp op) {
   if (hasSVAttributes(op))
-    emitError(op, "SV attributes emission is unimplemented for the op");
-
-  startStatement();
-  StringRef prefix = "";
-  if (op->hasAttr("doNotPrint")) {
-    prefix = "// ";
-    ps << "// This interface is elsewhere emitted as a bind statement."
-       << PP::newline;
-  }
+    return emitError(op, "SV attributes emission is unimplemented for the op");
```

Comment:
There's a difference in behaviour that I'm not sure about. Should we early return with failure (new behaviour), or emit the error and keep going (old behaviour)? Maybe this should be a warning instead of an error?